src/main/java/com/ss/mqtt/broker/model/topic/TopicFilter.java

```diff
@@ -0,0 +1,39 @@
+package com.ss.mqtt.broker.model.topic;
+
+import org.jetbrains.annotations.NotNull;
+
+public class TopicFilter extends AbstractTopic {
+
+    public TopicFilter(@NotNull String topicName) {
```

Comment:
could you extract this validation to TopicUtils and check it before allocation of this object?